src/components/TodoForm/TodoForm.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './TodoForm.css';
+
+export class TodoForm extends React.PureComponent {
+  state = {
+    enteredValue: '',
+    selectedUser: '',
+    showValueError: false,
+    showUserError: false,
+    userID: 3,
+  }
+
+  changeHandler = (event) => {
+    const { name, value } = event.target;
+
+    this.setState({ [name]: value });
+  }
+
+  addHandler = () => {
+    const { addTodo } = this.props;
+    const { enteredValue, selectedUser, userID } = this.state;
+
+    if (!enteredValue.trim() || !selectedUser) {
+      if (!enteredValue.trim()) {
+        this.setState({ showValueError: true });
+        setTimeout(() => (this.setState({ showValueError: false })), 2000);
+      }
+
+      if (!selectedUser) {
+        this.setState({ showUserError: true });
+        setTimeout(() => (this.setState({ showUserError: false })), 2000);
+      }
+
+      return;
+    }
+
+    this.setState(prevState => ({
+      enteredValue: '',
+      selectedUser: '',
+      showValueError: false,
+      showUserError: false,
+      userID: prevState.userID + 1,
```

Comment:
you need to find a correct userId in users by user name, it's not a random number